geth/install.py

```diff
@@ -94,13 +106,16 @@ def ensure_path_exists(dir_path):
     return False
 
 
-def ensure_parent_dir_exists(path):
+def ensure_parent_dir_exists(path: str) -> None:
     ensure_path_exists(os.path.dirname(path))
 
 
 def check_subprocess_call(
-    command, message=None, stderr=subprocess.STDOUT, **proc_kwargs
-):
+    command: list[str],
+    message: str | None = None,
```

Comment:
I think you can use `Optional[str]` in cases like this.

Reply:
This is a result of using the `from __future__ import annotations` - It allows the new (>=py310) PEP 604 syntax instead of `Union`